pkg/security/security_profile/activity_tree/activity_tree_proto_enc_v1.go

```diff
@@ -80,7 +80,7 @@ func processNodeToProto(p *model.Process) *adproto.ProcessInfo {
 		Pid:         p.Pid,
 		Tid:         p.Tid,
 		Ppid:        p.PPid,
-		Cookie:      p.Cookie,
+		Cookie:      uint32(p.Cookie),
```

Comment:
We should probably update the protobuf definition as well

Reply:
yes, probably, not sure to understand why we expose that field yet